code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -215,16 +191,22 @@ rcd light flash thingy on matter drain
 	user_type = USER_TYPE_MALFAI
 	panel = MALFUNCTION
 	charge_type = Sp_CHARGES
-	charge_max = 3
+	charge_max = 1
 	hud_state = "blackout"
 	override_base = "grey"
 
 /spell/aoe_turf/blackout/cast(var/list/targets, mob/user)
 	for(var/obj/machinery/power/apc/apc in power_machines)
-		if(prob(30*apc.overload))
-			apc.overload_lighting()
-		else
-			apc.overload++
+		apc.overload_lighting()
+	to_chat(world, radio_list.len)
+	for(var/obj/item/device/radio/R in radio_list)
+		R.scramble_message = 1
+	for(var/obj/item/device/rcd/matter/engineering/rcd in rcd_list)
+		rcd.disabled = 1
+	for(var/obj/item/mecha_parts/mecha_equipment/tool/red/red in red_tool_list)
```

Comment:
![image](https://user-images.githubusercontent.com/17928298/94008997-3679a280-fd7a-11ea-92c0-15e6cc79bdb5.png)

Reply:
The RED still uses an RCD insternally so making them break themselves can potentially solve this, too.